### DISCUSSION TITLE
Element Visibility in Design Settings

### DISCUSSION THREAD

**Discussion by ghost** [templaza/astroid-framework#655] [Q&A]
Is someone kind enough to explain to me how this works?

I can choose between "Current & all linked pages" and "Current menu item only". But I can't tell the difference.

**Comment by Chacapamac** [comment]
My guess is that "Current & all linked pages" mean the page and all page linked to it by page navigation
"Current menu item only" mean probably only the page call by the menu item, not the linked page.

I am maybe wrong with that....
